### ISSUE TITLE
Support Accessibility regarding Labels, Roles, Tooltips and Focus Ordering

### ISSUE THREAD

**Issue by ansarmie** [SeleniumHQ/selenium-ide#963]
## 🐛 Bug Report

Fulfill the **W3 Standards** **1.3.1 - Info and Relationships**, **2.1.1 - Keyboard**, **2.4.3 - Focus Order**, **4.1 - Compatible**, and the **European EN Standard 301 549** regarding **11.5.2.13 - Tracking of focus and selection attributes** (Recommendation) to support the use of the application by people with Hard of sight and Keyboard or Screen Reader users. Pull request #973

More info here:

- [Guideline 1.3 Adaptable](https://www.w3.org/TR/WCAG21/#adaptable)
- [Understanding Success Criterion 1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)
- [Guideline 2.1 Keyboard Accessible](https://www.w3.org/TR/WCAG21/#keyboard-accessible)
- [Understanding Success Criterion 2.1.1: Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)
- [Guideline 2.4 Navigable](https://www.w3.org/TR/WCAG21/#navigable)
- [Understanding Success Criterion 2.4.3: Focus Order](https://www.w3.org/WAI/WCAG21/Understanding/focus-order.html)
- [Guideline 4.1 Compatible](https://www.w3.org/TR/WCAG21/#compatible)
- [Understanding Success Criterion 4.1.2: Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html)
- [11.5.2.13 Tracking of focus and selection attributes](https://www.etsi.org/deliver/etsi_en/301500_301599/301549/02.01.02_60/en_301549v020102p.pdf)

## To Reproduce

- The log tab is not providing time in a way that a screen reader can understand. One possibility is using `<time>`.
- Several controls are not announcing their names or labels when focused.
- Several controls are not providing their role and the Screen Reader is not telling the user their purpose (Select, Menu, List, Input, Button) when focused.
- The Add New Test Case window is not announcing, in the Screen Reader, it's contents, description or purpose (in general, this happens to every Dialog)
- The Tooltip of every button is not appearing on focus.
- When editing the project name in the popup, the tabs doesn't go to OK.

## Expected behavior

- The Screen Reader should read time in human recognizable form.
- Every control should have both the name and a role (Selector, Input, Label) specified.
- Every popup should announce it's contents.
- The tooltip should on focus and disappear on blur. It should also be dismisable when the user press ESC.
- The focus order should follow the screen layout and the user localization if that's enabled. For the English language it's from left to right and from top to bottom.

## Project file reproducing this issue (highly encouraged)

This is a issue with the UI, not of a specific Selenium scenario.

## Environment

OS: Windows 10
Selenium IDE Version:  3.16.1
Browser: Google Chrome
Browser Version: 80.0.3987.116 (Official Build) (64-bit)

**Comment by tourdedave** [comment]
Fixed in [v3.17.0](https://github.com/SeleniumHQ/selenium-ide/releases/tag/v3.17.0)
